colmap.cpp

```diff
@@ -53,6 +53,12 @@ InputData inputDataFromColmap(const std::string &projectRoot){
             cam->fy = readBinary<double>(camf);
             cam->cx = readBinary<double>(camf);
             cam->cy = readBinary<double>(camf);
+        }else if (model == SimpleRadial){
+            cam->fx = readBinary<double>(camf);
+            cam->fy = cam->fx;
+            cam->cx = readBinary<double>(camf);
+            cam->cy = readBinary<double>(camf);
+            cam->k = readBinary<double>(camf);
```

Comment:
`k` is not a valid parameter. See https://github.com/pierotofy/OpenSplat/blob/main/input_data.hpp#L12

Reply:
Understood, I was copying verbatim from the colmap parameter list. Would 'k1' be a reasonable substitute in this situation for what I designated as 'k'? I don't fully understand what the other parameters represent past the 'fx' and 'fy' focal length parameters.